proxy/mempool/mempool_periodic_task_gas_price.py

```diff
@@ -60,5 +64,22 @@ def _process_error(self, _: MPGasPriceRequest) -> None:
         pass
 
     async def _process_result(self, _: MPGasPriceRequest, mp_res: MPGasPriceResult) -> None:
+        for token_list in mp_res.token_list:
+            chain_id_token_name = f'{token_list.chain_id}:{token_list.token_name}'
+
+            if chain_id_token_name not in self._min_executable_gas_prices:
+                self._min_executable_gas_prices[chain_id_token_name] = list()
+
+            if token_list.min_executable_gas_price > 0:
+                self._min_executable_gas_prices[chain_id_token_name].append(token_list.min_executable_gas_price)
+
```

Comment:
Better to store the reference to the list in the variable and use it, instead of requesting the list by the key each time:
```
min_gas_price_list = self._min_executable_gas_prices[token_list.chain_id]
min_gas_price_list.append(token_list.min_executable_gas_price)
while self._min_executable_gas_prices_count <= len(min_gas_price_list):
    min_gas_price_list.pop(0)
...
```

Reply:
No problem. Done.